src/app/zones/views/ZoneDetails/ZoneDetailsForm/ZoneDetailsForm.test.tsx

```diff
@@ -43,13 +42,13 @@ describe("ZoneDetailsForm", () => {
       </Provider>
     );
 
-    wrapper.find("button[data-testid='cancel-action']").simulate("click");
+    await userEvent.click(screen.getByTestId("cancel-action"));
```

Comment:
Could you use a text label instead of a testid?

Reply:
nvm i did it